### ISSUE TITLE
Access to RawString through automerge-repo?

### ISSUE THREAD

**Issue by joshuahhh** [automerge/automerge-repo#326]
My code, which is primarily automerge-repo-based, accessed RawString through a '@automerge/automerge/next' import. At one point, I noticed that I didn't have '@automerge/automerge' in my package.json, which I reflexively consider bad practice, so I added it. This actually broke my code, I think because automerge-repo used a different version of automerge. (This was the root cause of https://github.com/automerge/automerge/issues/891, fwiw.)

I want to flag this as a potential issue. My guess is that the best approach would be for automerge-repo to be a "one stop shop" – just import it and you'll get everything you need in a nice consistent bundle, including access to RawString. If it's better to access RawString through a separate automerge import, I wonder what approach would make sure that versions are in sync, and/or provide helpful error messages if they get out of sync.

(Thanks!)

**Comment by pvh** [comment]
I agree! I'd like to go through all of Automerge-Core and make sure we export any types and functions you might need as well as generally cleaning up the API. I think that might be a good small project to make a point release out of.
